ctapipe/reco/ImPACT.py

```diff
@@ -161,12 +169,17 @@ def initialise_templates(self, tel_type):
 
             self.prediction[tel_type[t]] = \
```

Comment:
avoid `\` for continuation lines, use parentheses. I would recommend running "black" on this file in fact.

Reply:
Done.